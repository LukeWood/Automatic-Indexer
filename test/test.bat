a.exe test.input test.cat test.out
