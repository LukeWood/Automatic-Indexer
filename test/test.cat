lorem = ipsum there
cat = meow this
bob = abc aa atc
zate = zack
