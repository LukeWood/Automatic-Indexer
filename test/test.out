: 1
aa: see bob
abc: see bob
about: 3
adipiscing: 4
amet: 4
anywhere: 2
appears: 1
atc: see bob
bob:  Not Found
book: 1
but: 2
capitalization: 2
cat: 1
condimentum: 4
consectetur: 4
dolor: 4
elit: 4
indexing: 3
individual: 3
ipsum: see lorem
is: 1, 2
lorem: 1, 4, 5
meow: see cat
need: 3
no: 1
not: 3
note: 1
of: 1
on: 1
one: 1
only: 3
page: 1
phasellus: 4
phrases: 3
present: 2
punctuation: 2
sentencebased: 2
sit: 4
still: 2
text: 1
that: 1, 2
the: 1
there: see lorem
this: see cat
to: 3
words: 3
worry: 3
you: 3
zack: see zate
zate:  Not Found
